### ISSUE TITLE
documentation doesn't explain managed credential scan setup

### ISSUE THREAD

**Issue by ykram** [tenable/pyTenable#123]
**Describe the bug**
In io, when creating a scan and passing a 'credentials' keyword in the scan.create method, the scan gets created on the tenable io side but there aren't any managed credentials added to the scan. Documentation is very lacking in this regard.

**To Reproduce**
call tio.scans.create() with 

credentials={'Host': {'SSH': [{'auth_method': 'public key', 'username': 'tenable', 'elevate_privileges_with': "Nothing", 'private_key': 'tenable' }]}}

**Comment by SteveMcGrath** [comment]
The credentials input will eventually be refactored once I have a more concrete way to validate inputs.  however for the SSH keys, you'll generally need to run a files.upload first with the keyfile, then use the resulting filename returned for the private_key param.  See if that yields better results.

Sadly, pyTenable is just passing through the dictionary in this case, and not validating.

**Comment by ykram** [comment]
Thanks for the update. In the event that a key was already uploaded and is listed under the "managed credentials" section, would I still need to upload the key or is there a way to specify that managed credential ID when initiating the scan? Is there an example of this I could potentially look at?

**Comment by ykram** [comment]
I think I got it for now. In case anybody else wants to use the SDK to setup credentialed scanning with manged credentials, the steps I took were:

1. grab the "uuid" of the managed credential from the `/credentials` endpoint
2. when specifying the scan options with` scans.create()`, I set the credentials parameter specifically to:
`credentials={'Host':{'SSH':[{'id': 'managed-scan-uuid-here'}]}`

Hope this helps anybody else who be trying to do the same thing.
